qdrant-landing/content/articles/vector-search-production.md

```diff
@@ -0,0 +1,517 @@
+
+This not only speeds up query throughput for large-scale datasets, but also cuts hardware costs and storage overhead. While Scalar Quantization is a midrange compression alternative, Binary quantization is more drastic, so be sure to test your accuracy requirements for each thoroughly.
+
+When using [**quantization**](https://qdrant.tech/documentation/guides/quantization/), you can store only the compressed vectors in memory while leaving the original floating-point versions on disk for reference. This approach dramatically lowers RAM consumption—since quantized vectors take far less space—yet still allows you to retrieve full-precision vectors if needed for downstream tasks like re-ranking.
+
+>**Sidenote:** You can always enable `async_io` scorer when the linux kernel supports it and if you have `on_disk` vectors.
+
+||
+|-|
+|**Read More:** [**Quantization Documentation**](https://qdrant.tech/documentation/guides/quantization/)|
+
+## 2. How do I Ingest and Index Large Amounts of Data?
+![vector-search-production](/articles_data/vector-search-production/vector-search-production-2.jpg)
+
+||
+|:-:|
+|**"When I try to import a huge dataset, everything grinds to a halt."**|
+
+❓ **Use Case:** A fintech team ingested 500 million transaction records. Performance was fine initially but collapsed within an hour. 
+
+They had left HNSW indexing enabled, so every insert triggered a full index update. Unfortunately, their CPU usage soared, and other services timed out.
+
+> ✅ On a case-by-case basis, we recommend **disabling building the HNSW index during large uploads** to improve ingestion and indexing speed. 
+
+Once all records are inserted, you can rebuild the index in a single pass. Consider a specialized ingestion pipeline that batches writes and schedules indexing during low-traffic windows. If you don't have such low-traffic windows, you can tune the `indexing_threshold` to find a balance between receiving updates without triggering indexation, and keeping the collection indexed.
+
+||
+|-|
+|**Read More:** [**Configuring the Vector Index**](https://qdrant.tech/documentation/concepts/indexing/#vector-index)|
+
+### Other Solutions to Alleviate Indexing Bottleneck
+
+✅ **Increase indexing threads:** If you're using more than 16 cores, consider explicitly increasing the number of indexing threads. 
+
+> By default, Qdrant uses around 16 threads for indexing, but if you notice your CPU isn't being fully utilized during indexing, you can increase this number.
+
+✅ **Optimize batch size and concurrency:** Keep your batch size at the default (around 100) and instead increase the number of concurrent processes. Running 50-60 concurrent processes can significantly improve upload performance. Using just one or two processes won't allow you to see the true performance potential.
```

Comment:
Not so sure about this one. If you already use batching with wait=false it's likely that you already saturate upserting, in which case it will not be much faster with more parallel processes.

I think just stating 'use batching' is good enough.